src/ensembl/src/shared/components/help-popup/HelpPopupBody.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import styles from './HelpPopupBody.scss';
+
+export type HelpVideo = {
+  title: string;
+  description: string;
+  url: string;
+};
+
+type ArticleSummary = {
+  title: string;
+  slug: string;
+  path: string;
+};
+
+export type HelpArticle = {
+  path: string;
+  slug: string;
+  body: string;
+  videos: HelpVideo[];
+  related_articles: ArticleSummary[];
+};
+
+type Props = {
+  article: HelpArticle;
+  onArticleChange: (slug: string) => void;
+};
+
+const HelpPopupBody = (props: Props) => {
+  const { article } = props;
+
+  const videos = article.videos.map((video) => (
```

Comment:
Are we displaying multiple videos in a single popup?

Reply:
I assumed that yes. Need to ask Andrea. I hope she will want to update the design of the popup anyway, because currently the video is unusably small — user would always have to switch it to full screen if they want to see what's going on.

Reply:
@msmanoj I discussed the popup with Andrea, and we came to the following conclusions:

- Andrea needs to rethink the design of the contextual popup — specifically, what should happen when user clicks on a related article or a related video link.
- Until she does, there isn't much point in trying to style the popup window one way or the other; because we don't know what the final design is going to look like.
- there have already been changes, e.g. this design, which you likely used when you worked on contextual help:

<img src="https://user-images.githubusercontent.com/6834224/89314374-7c1ac880-d671-11ea-8bb4-b933d82280a5.png" width="400">

has been changed to this (what I based my implementation on):

<img src="https://user-images.githubusercontent.com/6834224/89314497-a2d8ff00-d671-11ea-830d-8d7af44e5a9e.png" width="400">

But even the second one is not final.

So I wouldn't worry about the number of allowed videos in the middle column. But I will try to avoid the flickering of the popup window when changing the article.